docs/quickstarts/tanstack-start.mdx

```diff
@@ -39,15 +39,15 @@ description: Learn how to use Clerk to quickly and easily add secure authenticat
 
   <CodeBlockTabs options={["npm", "yarn", "pnpm"]}>
     ```bash {{ filename: 'terminal' }}
-    npm install @clerk/tanstack-start@beta
+    npm install @clerk/tanstack-start
     ```
 
     ```bash {{ filename: 'terminal' }}
-    yarn add @clerk/tanstack-start@beta
+    yarn add @clerk/tanstack-start
     ```
 
     ```bash {{ filename: 'terminal' }}
-    pnpm add @clerk/tanstack-start@beta
+    pnpm add @clerk/tanstack-start
```

Comment:
We don't actively publish the package under the `@beta` tag so the current beta tagged version is outdated compared to `latest`. We don't use that tag for other packages that are still Beta (e.g. Elements) so this changes it so that people installed the latest version